docs/openapi/components/schemas/common/PPQ368NoticeOfArrival.yml

```diff
@@ -0,0 +1,290 @@
+$linkedData:
+  term: PPQ368NoticeOfArrival
```

Comment:
Typically we prefix with the schema authority. I wonder if we could include that here too - USDA, I suppose?